packages/instrumentor/edgeIdStrategy.ts

```diff
@@ -21,6 +21,18 @@ import process from "process";
 
 import { fuzzer } from "@jazzer.js/fuzzer";
 
+if (process.listeners) {
+	// "signal-exit" library imported by "proper-lockfile" inserts listeners for all important signals, such as SIGALRM and SIGINT (see https://github.com/tapjs/signal-exit/blob/39a5946d2b04d00106400c0dcc5d358a40892438/signals.js)
```

Comment:
Could you break up the long comment lines?

Reply:
Oh, I thought that the linter would take care of that. Good to know it doesn't!

Reply:
AFAIK only in markdown files.